app/javascript/components/wifi.jsx

```diff
@@ -5,6 +5,8 @@ import gql from 'graphql-tag';
 import { WhiteSubTitle, WhiteTitleLarge } from '@components/typography';
 import { Row } from '@components/row';
 import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+import WifiIcon from '@assets/images/wifi_icon.svg';
```

Comment:
maybe we name this `wifi_name.svg` and `wifi_password.svg` since that's the name of the text for these icons?

Reply:
@fallynlogan, once you're finished with the updated designs, all you need to do is replace these svgs with your new files (and possibly update the width and height to make sure they look good).

Reply:
@jennyckaplan sounds good I will work on that today!